shuffle/cli/src/account.rs

```diff
@@ -17,13 +17,21 @@ use generate_key::load_key;
 use shuffle_transaction_builder::framework::encode_create_parent_vasp_account_script_function;
 use std::path::PathBuf;
 
-pub fn handle(project_dir: PathBuf, account_key_path: PathBuf) -> Result<()> {
+pub fn handle(project_dir: PathBuf, account_key_path: Option<PathBuf>, random : bool) -> Result<()> {
+    let new_account_key;
+    if random {
+        new_account_key = generate_key::generate_key();
+    } else {
+        if account_key_path == None {
```

Comment:
don't need to compare to none; I believe you can just do:
```
load_key(account_key_path.expect("Enter the account key path or use the -r random flag"));
```

Reply:
We return result, so perhaps we should get rid of this part and then below do:
`new_account_key = load_key(account_key_path?);`